Docs/DafnyRefMd/Makefile

```diff
@@ -0,0 +1,3 @@
+all:
+	pandoc -t pdf -o DafnyRef.pdf --pdf-engine=xelatex --toc DafnyRef.md
```

Comment:
We should get #605 merged in and use it here - I created that precisely to get the syntax highlighting to work for pandoc.